src/diffusers/pipelines/stable_diffusion/__init__.py

```diff
@@ -44,6 +44,7 @@ class StableDiffusionPipelineOutput(BaseOutput):
     from .pipeline_stable_diffusion_inpaint_legacy import StableDiffusionInpaintPipelineLegacy
     from .pipeline_stable_diffusion_instruct_pix2pix import StableDiffusionInstructPix2PixPipeline
     from .pipeline_stable_diffusion_latent_upscale import StableDiffusionLatentUpscalePipeline
+    from .pipeline_stable_diffusion_pix2pix_zero import StableDiffusionPix2PixZeroPipeline
```

Comment:
```suggestion
```

Reply:
Can you add it instead below 
```
from .pipeline_stable_diffusion_depth2img import StableDiffusionDepth2ImgPipeline
```

to only import it when transformers version is higher than 4.26.0

Reply:
Because of https://github.com/huggingface/diffusers/pull/2334/files#r1106862799

Reply:
And can you also add the pipeline here: https://github.com/huggingface/diffusers/blob/91925fbb761d944d54271660c4c3cffee55798fa/src/diffusers/utils/import_utils.py#L412